ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/featuredcategorylist/v1/featuredcategorylist/clientlibs/site/css/featuredcategorylist.css

```diff
@@ -0,0 +1,29 @@
+.cmp-categorylist__anchor {
+    text-align: center;
+}
+
+.cmp-categorylist__content {
+    display: grid;
+    grid-gap: 3rem 2rem;
+    grid-template-columns: repeat(auto-fit, 7rem);
+    justify-content: center;
+}
+
+.cmp-categorylist__image {
```

Comment:
That kind of styling is not very nice, since we use a background with an image on top. The same can be achieved on the `img` tag directly using https://www.w3schools.com/css/css3_object-fit.asp.

However, that's how it was built in HTL, so let's stick to it for now.